crates/sui-core/src/authority_aggregator.rs

```diff
@@ -65,34 +66,20 @@ pub type AsyncResult<'a, T, E> = BoxFuture<'a, Result<T, E>>;
 
 #[derive(Clone)]
 pub struct TimeoutConfig {
-    // Timeout used when making many concurrent requests - ok if it is large because a slow
-    // authority won't block other authorities from being contacted.
-    pub authority_request_timeout: Duration,
     pub pre_quorum_timeout: Duration,
     pub post_quorum_timeout: Duration,
 
-    // Timeout used when making serial requests. Should be smaller, since we wait to hear from each
-    // authority before continuing.
-    pub serial_authority_request_timeout: Duration,
-
     // Timeout used to determine when to start a second "serial" request for
-    // quorum_once_with_timeout. This is a latency optimization that prevents us from having
-    // to wait an entire serial_authority_request_timeout interval before starting a second
-    // request.
-    //
-    // If this is set to zero, then quorum_once_with_timeout becomes completely parallelized - if
-    // it is set to a value greater than serial_authority_request_timeout then it becomes
-    // completely serial.
+    // quorum_once_with_timeout. If this is set to zero, then
+    // quorum_once_with_timeout becomes completely parallelized.
     pub serial_authority_request_interval: Duration,
 }
 
 impl Default for TimeoutConfig {
     fn default() -> Self {
         Self {
-            authority_request_timeout: Duration::from_secs(60),
             pre_quorum_timeout: Duration::from_secs(60),
-            post_quorum_timeout: Duration::from_secs(30),
-            serial_authority_request_timeout: Duration::from_secs(5),
+            post_quorum_timeout: Duration::from_secs(7),
```

Comment:
Out of curiosity, how did we come up with `7` seconds?

Reply:
this is the latency when the first submitter fails to submit in narwhal. Do you recommend another value?